11/umbraco-cms/extending/property-editors/tracking.md

```diff
@@ -6,9 +6,9 @@ description: >-
 
 # Tracking References
 
-Property editors can be extended further to track entity references that may be selected or referenced inside the property editor. For example in the core of the CMS we have added this to several property editors.
+Property editors can be extended further to track entity references that may be selected or referenced inside the property editor. For example in the core of the CMS we have added this to numerous property editors.
 
-A good example of this is the Media Picker where the CMS stores a reference to what media item was picked and thus allows to see what content nodes are using a media item. This avoids it being accidentally deleted if it is being used.
+A good example of this is the Media Picker. The CMS stores a reference to the selected media item, enabling the identification of content nodes that use that particular media item. This avoids it being accidentally deleted if it is being used.
```

Comment:
Am not sure if identification is a right word to use here and I think the sentence "The CMS stores a reference to what media item was picked and thus allows to see what content nodes are using a media item" is quite ok by itself. This sentence is shortly trying to say that once you click on a media item you can see the references of where it is used in the content nodes, so maybe the suggestion bellow could be better to define this?
```suggestion
A good example of this is the Media Picker. The CMS stores a reference to the selected media item, enabling to see what content nodes are using the selected media item. This avoids it being accidentally deleted if it is being used.
```